setup-wizard.yml

```
@@ -3,10 +3,10 @@ fields:
   - title: Execution Client Config Mode
```

```suggestion
  - id: config_mode
```
despite it not mattering in a yaml Im gonna just keep the style consistent making the id the first entry for each option, just flipping the title and id here before i push to the other clients